src/customizations/volto/components/theme/View/EventDatesInfo.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { List } from 'semantic-ui-react';
-import moment from 'moment';
 import cx from 'classnames';
 import { RRule, rrulestr } from 'rrule';
```

Comment:
Hai provato per caso a fare lazy loading anche di rrule? Anche se poi sarebbe da fare anche in volto core, mi interessa capire se dà noie o se si va via lisci

Reply:
non ci ho provato, l'ho lasciato li per ultimo 
oggi ci provo.. ma dato che non è importato lazy neanche in volto nelle ricorrenze, non so quanto cambi importarlo lazy su io-comune

Reply:
Sì esatto, come dicevo era giusto per capire se dà noie, che in caso poi appena possibile lo andiamo a fare anche nel core.

Reply:
@pnicolli ho fatto il lazy loading di rrule su io-comune. Per come era stato usato su io-comune è stato semplice fare il lazy loading. Proviamo a farlo anche su volto